### PR TITLE
[MindMap] Add new role and responsibility

### PR DESCRIPTION
**Pull request by jlprat**
Adds Individual Contributor as a role within the OSPO
Adds Upstream OS contributions as responsibilities
Fixes some linting problems in the md

### CODE REVIEW

**Comment by jlprat** [comment]
The linting problems were mostly either extra spaces at the end of the line, or missing empty lines around titles and lists

**Comment by jlprat** [comment]
All good, I'll generate the files later today!

File: ospo-mindmap/Img/ospomindmap.html [hunk 1]
```diff
@@ -16,13 +16,13 @@
   height: 100vh;
 }
 </style>
-<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/markmap-toolbar@0.2.0/dist/style.css">
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/markmap-toolbar@0.2.0/dist/style.min.css">
```
Comment (jlprat) [inline ospo-mindmap/Img/ospomindmap.html:19]: I don't know why my vesion uses the `min` file, but this should be a good thing, smaller file size with full functionality

Comment (anajsana) [reply ospo-mindmap/Img/ospomindmap.html:19]: Ok! I've also updated the index.html within the public folder with the new html

Comment (jlprat) [reply ospo-mindmap/Img/ospomindmap.html:19]: Thanks!

**Pull request merged**